src/EFCore.PG/ValueGeneration/Internal/NpgsqlSequenceValueGeneratorFactory.cs

```diff
@@ -31,15 +32,19 @@ public class NpgsqlSequenceValueGeneratorFactory : INpgsqlSequenceValueGenerator
             Check.NotNull(rawSqlCommandBuilder, nameof(rawSqlCommandBuilder));
             Check.NotNull(sqlGenerator, nameof(sqlGenerator));
 
-            _rawSqlCommandBuilder = rawSqlCommandBuilder;
             _sqlGenerator = sqlGenerator;
         }
 
         /// <summary>
         ///     This API supports the Entity Framework Core infrastructure and is not intended to be used
         ///     directly from your code. This API may change or be removed in future releases.
         /// </summary>
-        public virtual ValueGenerator Create(IProperty property, NpgsqlSequenceValueGeneratorState generatorState, INpgsqlRelationalConnection connection)
+        public virtual ValueGenerator Create(
+            IProperty property,
+            NpgsqlSequenceValueGeneratorState generatorState,
+            INpgsqlRelationalConnection connection,
+            IRawSqlCommandBuilder rawSqlCommandBuilder,
```

Comment:
Do we need to add a null check?

Reply:
In fact, upstream has no checks whatsoever - because this is an internal API. Am going to do the same...